spread/project.go

```diff
@@ -139,6 +139,14 @@ type System struct {
 
 	Priority OptionalInt
 	Manual   bool
+
+	// Specify that the disk should be attached using the "virtio" backend
+	// in QEMU. Only supported by the qemu backend.
+	VirtioDisk bool `default:"false" yaml:"virtio-disk"`
```

Comment:
This is fairly specifc to qemu right now, but the `System` struct is used across all backends. So a more generic mechanism sounds more useful, so maybe a string option like: `disk-driver` or `virtual-disk-driver` or similar and then the user could set it to `virtio-disk`  on qemu and on different backends there could be a different value here (same for the network of course).

I also wonder if this could become a map even, so it would be something like:
```
virtual-drivers:
 disk: virtio-disk
 network: virtio-net
```
and depending on the backend it would be interpreted differently.

Reply:
Great suggestion. Should this fail on systems that do not implement this, or should they just ignore the options that are specified?